compiler-cli/src/build.rs

```diff
@@ -10,14 +10,19 @@ use gleam_core::{
 
 use crate::{
     build_lock::BuildLock,
-    cli,
-    dependencies::UseManifest,
+    cli, dependencies,
     fs::{self, get_current_directory, get_project_root, ConsoleWarningEmitter},
 };
 
 pub fn download_dependencies(telemetry: impl Telemetry) -> Result<Manifest> {
     let paths = crate::find_project_paths()?;
-    crate::dependencies::download(&paths, telemetry, None, Vec::new(), UseManifest::Yes)
+    dependencies::download(
+        &paths,
+        telemetry,
+        None,
+        Vec::new(),
+        dependencies::DependencyManagerConfig::default(),
```

Comment:
This is much less clear than before- please explicitly state each item of config. The programmer should be able to tell what the desired behaviour is at each call-site. This will mean removing the default implementation.

Reply:
Done